php/WP_CLI/CommandWithTranslation.php

```diff
@@ -125,6 +125,95 @@ public function install( $args, $assoc_args ) {
 	}
 
 	/**
+	 * Updates the active translation of core, plugins, and themes.
+	 *
+	 * [--dry-run]
+	 * : Preview which translations would be updated.
+	 *
+	 * @subcommand update
+	 */
+	public function update( $args, $assoc_args ) {
+
+		// Ignore updates for the default locale.
+		if ( 'en_US' == get_locale() ) {
+			\WP_CLI::line( "Translations updates are not needed for the default locale." );
+
+			return;
+		}
+
+		wp_version_check();  // Check for Core updates.
+		wp_update_themes();  // Check for Theme updates.
+		wp_update_plugins(); // Check for Plugin updates.
+
+		$updates = wp_get_translation_updates(); // Retrieves a list of all translations updates available.
```

Comment:
It just gets a list of translation updates from the database.

The functions `wp_version_check`, `wp_update_themes` and `wp_update_plugins` checks for updates (which include translations updates) and store it in the database. After that, the function `wp_get_translation_updates` gets the info and return it as an array.

Translations updates are performed by the method `upgrade` (single update) or `bulk_upgrade` (all updates at once) of the WordPress class `Language_Pack_Upgrader`.